tensorflow/java/src/main/native/saved_model_bundle_jni.cc

```diff
@@ -20,14 +20,36 @@ limitations under the License.
 #include "tensorflow/java/src/main/native/exception_jni.h"
 #include "tensorflow/java/src/main/native/saved_model_bundle_jni.h"
 
+
 JNIEXPORT jobject JNICALL Java_org_tensorflow_SavedModelBundle_load(
     JNIEnv* env, jclass clazz, jstring export_dir, jobjectArray tags,
-    jbyteArray run_options) {
+    jbyteArray run_options, jbyteArray session_options) {
   TF_Status* status = TF_NewStatus();
   jobject bundle = nullptr;
 
   // allocate parameters for TF_LoadSessionFromSavedModel
   TF_SessionOptions* opts = TF_NewSessionOptions();
+
+
+  TF_Buffer* csession_options = nullptr;
```

Comment:
The formatting here seems to be different from the conventions used in the rest of the file (e.g., two-spaces instead of 4). Could you remove that inconsistency?

(In general, the native code should [follow the Google style guide](https://google.github.io/styleguide/cppguide.html))